### ISSUE TITLE
aria-actions attribute

### ISSUE THREAD

**Issue by janewman** [WebKit/standards-positions#686]
### WebKittens

@cookiecrook

### Title of the proposal

aria-actions addition to the ARIA spec

### URL to the spec

https://github.com/w3c/aria/pull/1805

### URL to the spec's repository

https://github.com/w3c/aria

### WebKit Bugzilla URL

https://bugs.webkit.org/show_bug.cgi?id=306476

### Description

A new aria attribute that references a list of elements that should be exposed as secondary actions.

**Comment by bkardell** [comment]
In the triage meeting today we decided to propose that we add a support label here and close the issue in a week
